### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by MarkYegon7**
In this project, I was able to:

- Use correct GitHub flow principles.
- Use linters.
- Use Flexbox and CSS Grid to align elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,14 @@
...
+            <ul class="nav-content">
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+              </ul>
+        </nav>
+
     </header>
 
     <section class="landing-page all-body" id="body-cover">
```
Comment (Zuhaib042) [inline index.html:26]: - [ ] Kindly consider setting `<div class="ctn">` div in the center of the page as shown in the figma design that in the desktop screen size, it has equal margins on the left and right side. we need to stick to the figma design as much as possible.
your design
Screenshot 2023-07-13 at 10 17 28 PM

figma design
Screenshot 2023-07-13 at 10 17 58 PM

File: styles.css [hunk 2]
```diff
@@ -608,3 +608,220 @@ hr {
...
+    margin-top: 2rem;
+  }
+
+  .myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: auto;
+    padding-left: 5rem;
+    padding-right: 5rem;
+  }
```
Comment (Zuhaib042) [inline styles.css:767]: - [ ] In the about section on the figma design, the left block takes 40% of width of the container and the right block takes 60% width. but in your design both blocks takes 50% of width each. kindly consider fixing it as we need to stick to the figma design.
- [ ] Also kindly consider to add the button interactions styling in all the buttons in your web-page. you can apply these interactions by using `:hover and :active` pseudo-classes in hovering or pressed states.
your design
Screenshot 2023-07-13 at 10 24 00 PM

figma design
Screenshot 2023-07-13 at 10 23 42 PM

button styling guidelines
Screenshot 2023-07-13 at 10 33 02 PM

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @MarkYegon7 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Beautiful design
✅ Styling is on point

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MarkYegon7) [reply index.html:26]: Resolved the issue

File: styles.css (hunk 2 above)
Comment (MarkYegon7) [reply styles.css:767]: Solved both the container section and the button interactions

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @MarkYegon7 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done implementing the changes requested by the previous reviewer. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**